themes/helvetica/base.scss

```diff
@@ -2,6 +2,20 @@ body {
JDOQQOwrLAWL6d1xhgyZ3yyBjEkHQMiB7ljgs/gAO3nKaoUjQEvWyqdNBQTSfK0z3vk6c0AVhNwQs1a4/yZJCuz3EHYcWJh9RiRWZl2DGcrpETbHne0IMlMBhnlDgVWz3IdAmnj0NDePm9MUeAgeQOO1eAo5w07afMA0hU+d4CXaAw9YLUTvjjUR9rFgZSwx47z9JSmZBeNxUxIa02NUauD08HJoVWk6R9FD3CXOGn7OpMkhvDJ0EqoYZyxBfUKk3ayRqV8kFO1ceCGNkzICFXaQ0jWO/ZBH6RC2sKAppZTBpUfRdqOSMb+YS+f4IXSCKDK1Px5peXNChoIBAiWkBb+dcV/HnHY/AICACH5BAQFAAAALBcAFwA0ADUAAAj/AP8JHEjwnz9/3g4qXMjwoLcTvk4kZFiwokWBDhWyOHGCRcOF3i7ciYNCD4uJCy+qbGgpVpECwjx+9Dcrjs04TXCgTKmyYMNJAYoILcJgJgtWN21Cm+mv58CPWYYKJbNT4RZiSeOoYdq058yoUovM+fgm650TXLtaZIrjXFgEVVksyzqs6se1TL31C2ugB8MTn7LmSotQLca0+iKF3cawU9Y4pggXfkqYRZmwAdA6lJb1k8yP3kJnPEx4StgiIjQOy4rMrr8T9BQj0Ny0MOET+8KikbllZNKtDFlUOkenOJ0wCXdu3PK54YTTKg6qMUv7IA47DowXd4BDYfJuCQ6w/4LWXOGJAmFJHZRDF6VwBdqNt9G5sN2B+/fNWKl+sFFY5P5wkRUu5mGTXXzFFcCfKvg1OMw01W0R1FDCKFTDTQtoNkcBCBp3jg4N1dPgiAt0o4tCpUhRhAEgoOSNMfVMI5Mj8XRIBzx48HdQBCP2uIA2tK1QnncHnZCBjQYM4tpBhrDS44gELPkRAx3Cs4iODZ0AjZNP3qeFZAfpgGAkU0iZZTLAdGkImAjtIx8pWN6mR5oNJrAFm69hI8UuYJh5Gyo13JcAE3g6JFqhM3lzxQPuIOroo3iuEEUURggAQwsd+dlQaJoytUQdoIYKKgmOqMBAKY0megI1IiBAxjYrPP+agqi0ikpCCyCt2uo+RPTaaxhDVlbrsKC64A8LAuzq67K+TuFoFsTWOgQLIjBrra/GItqCI9GKygAP117rwSSPsgCDAKUwoMIQs4pKzgkuhMssGaVAatQkK/ABg0d8yOtBP8IIEGxaXTDBRBdseiPBsv8KM0dHC7HAxSxfpnWFCRibEAGb4UzRSMAQZ1lPDSSHkBYSGWMcQaeSeaMNyTAjrFE4C9GQMsZasJyWITDDLEaRjARSThYKzXIzzjrPdEbPMR8kTiBQBxKdP+5oc7QJe9jLwshMa1Nk1FCDoBAnNh+dtaPehMB0DbdodgLYQqNE9tWPOBrB2jVEdhALcAe5Ul0XZadMAyd3MaULAWur4SI9cOO60COBZ2wyRQaB1sHaHTTnzTZww9CQEkfT0BBlDGGxNgF3MiQB3CA2dHHKT1BOukNWrI3FR43AbcRMhWRMgxKjVQQSzz1P4xoVcDs70wYPMMoTXkVyXQM4AzsCd7ZgOmUbC2ogwcTA/qgDtzotG7aSzi7A7QhhCTlFkG2ElQI30YS57xP8TJ2ASdTlxJqW/dBLywrEgQkNtI4pAPRKWkJzEq7YLyAAIfkEBAUAAAAsFwAXADQANQAACP8A/wkcSPCfP3/eDipcyPCgt1NnTiVkWLCiRYEOFd44Me5Gw4XpLkBoZcvTuIkLL6psaGSXASkTPH70R6mVzVYotqBMqbJgwy0FDAg9N2XmDWA3bbqZ6a/nwI8qhEolI5Nhg5FJ/zBt2nNmVKlCBXzUk7RVk2JbuVpkiiMoWDw7/d24VJZV3I9rmXqjB9ZAgFcMTxwoKyctQrUY0+o41/cew05lW5kyfPip4Ruk+hY4odCbtLK2qjb0RjpjYsOD+hogoZFV2Ut3/Z1QxBgP54MGYzc8ca1vLJlXyy5leKOSAgfIHZTxuPPGqS3jmGZR7eLgn7IQbitsgC1DcuQZGnT/ljtgWIIxlKLvRtN30UFAZTGgvGGpwPfk+HCINpSgf38Ms5zSkAh9kZLQA2VlotAJ23h3H3JoaHdQPf5VyMo0Am7nllBDKOTETcNkqI8UDyangA4NXVDhiqwMgNZBRtTilwQopfPHBVZkOMQxJTqQgR0SKlTIikSa0U2G/rTwwkwTFWNDjwoMops/D2BA5IqwUXYQAyXGY5thp7gxxpX+aaGlP3w8SJRoaZ2gxypkPnCmN9ckF88iQWp5ypsrLqDLmf6cso0m10wxJWWnoEJAf+gwAahDN7D56EI3PPIAHJNmqimgLRhhhAB8rLDFCaU9KimgMAyh6qqrquBCFAK8//iRNycYQYIXH4DQgqa9sOorq1lssRCttnoBgAfIIhvDko+y8OuzqhZ1gw63HpvstchGkakL0P7qwg3CYCtusgxkukWv3a4aBQzjjkuGrM2uwIcAUTDgQha+ulDMFO1i+4ERm870whYtwLACC/6w2y4ZUEygz6lprcMFF+sA2si1DE8gwAmivfCAMYUY9ogPJPsQspbhRMGIIxuf+gI4FMQ8S1ohlEyyFocC6g0SMfdcsUbhLISEzTfnrOUDPfcsxkHFCLMGLL0oRAnRJF9hdFqcEJB0zD9TMcPXM1Tnjy5DU/1IwC90sDUFAxx0Cthfa6CQO2XbjIQSmnozy9oWZNpYDNwzwFIVJ3WXfHemhWidNAFnH3QD4DMg6c86hZOMBCd4MaULIWv/gZI3fQCOA0NKVO4DJQ2dxpA3cqwth3oOSQD4Cg2VTjQSqauuUARrEyIsQ0EAzsdHI9uMOk+Wdbb34mYOCLhYH11RNhI/J5SXQ0gnPcupwgBODVPuRBDBnwqtpFDaPZvAbENZAF7umU4d5s8LTAzABcIzuQC42GlZ71RuZ5oC4LJgGP/9D4CUMQLgVECZAxIkI2k5hey+toZdpcWBFdHSK6iwDXEMbysYNN9WvBGpQx0wIAA7)
```

Comment:
Any objections to extract this image to a static file that clients/browsers will cache?